classes/console/console.gd

```diff
@@ -37,19 +37,16 @@ func run_command(cmd: String):
 						path = "res://scenes/levels/castle/lobby/castle_lobby.tscn"
 					_:
 						path = ""
-				if path == "":
+				var err = OK
+				if path == "" or not File.new().file_exists(path):
```

Comment:
Is this the correct function?
Shouldn't ResourceLoader.exists() be used instead? Considering scenes are resources and godot might remove the source in the export.

Reply:
Yeah, `ResourceLoader.exists()` is better. Using the `File` class on internal resources won't work when running the exported executable. Where did the source use `File.new().exists()`?

Reply:
> Yeah, `ResourceLoader.exists()` is better. Using the `File` class on internal resources won't work when running the exported executable. Where did the source use `File.new().exists()`?

Lines 44 & 45

Reply:
I think that was something that Tryzen added but I forgot to check last time.
However, I don't think it was her fault. It turns out that the `scene` command has been using it, but nobody noticed because nobody uses that command in the exported version (it doesn't make much sense without context).

...Actually, that command doesn't work *at all*, because it requires the use of `/` for the scene paths. And you can't type that into the console, because it just closes when you try to. Whoops.